### DISCUSSION TITLE
How to modify camera settings at runtime:

### DISCUSSION THREAD

**Discussion by tlsduf** [Sixze/ALS-Refactored#563] [Q&A] [answered]
I want to change camera settings during gameplay, such as adjusting the FOV value for third person view. However, in alsCameraComponent.cpp, the only public class available besides the getters is void SetRightShoulder(bool bNewRightShoulder);. What should I do?

**Comment by ameaninglessname** [comment, marked as answer]
Create a new class, derive and customized it. Or searching alternatives in marketplace.

Previous discussion: #417

**Reply by tlsduf** [reply]
thanks for your Answer.
